pom.xml

```diff
@@ -977,8 +988,7 @@
                             <include>**/*Test.java</include>
                             <include>**/*TestSuite.java</include>
                         </includes>
-                        <argLine>-Xmx1024m -Xms128m -Dfile.encoding=UTF8 "-Dtesterum.packageDirectory=${project.basedir}/package"
-                        </argLine>
+<!--                        <argLine>-Xmx1024m -Xms128m -Dfile.encoding=UTF8 "-Dtesterum.packageDirectory=${project.basedir}/package"</argLine>-->
```

Comment:
This looks like a local experiment that got committed by mistake. Should we un-comment this line?